meta/group-info/ko.yaml

```diff
@@ -1,14 +1,11 @@
-outdated_since: 0522bc4c323f9a2b0c89a6b9582109a0608c6287
-outdated_translation: true
-
 separator: ", "
 alumni:
   roles:
     academy: osu!academy 설립자
     admin: 관리자
     chat: 챗 모더레이터
     dev: osu!dev
-    media: osu!media 설립자
+    media: osu!media 크리에이터
     support: 지원팀
```

Comment:
I said to update this file because of this

Reply:
reverted 😱